ocp_resources/virtual_machine_instance.py

```diff
@@ -117,16 +117,14 @@ def wait_until_running(self, timeout=TIMEOUT_4MINUTES, logs=True, stop_status=No
             TimeoutExpiredError: If VMI failed to run.
         """
         try:
-            self.logger.info(
-                f"VMI {self.name} status before wait: {self.instance.status.phase}"
-            )
             self.wait_for_status(
                 status=self.Status.RUNNING, timeout=timeout, stop_status=stop_status
             )
         except TimeoutExpiredError as sampler_ex:
             if not logs:
                 raise
-            self.logger.error(f"VMI {self.name} status: {self.instance.status.phase}")
+            if not self.exists:
+                self.logger.error(f"VMI {self.name} does not exist!")
```

Comment:
I think we can remove this now, with the new changes, we'll get the `ResourceNotFoundError` if `virt_launcher_pod` doesn't exist

Reply:
moved this to `wait_for_status()` (so if no `current_status`, it'll show proper log)
```
2023-08-31T15:03:01.402834 ocp_resources.resource INFO kind: VirtualMachineInstance api version: kubevirt.io/v1
2023-08-31T15:03:01.402999 ocp_resources VirtualMachineInstance INFO Wait for VirtualMachineInstance container-disk-vm-1693483207-3708382 status to be Running
2023-08-31T15:03:01.403144 ocp_resources.utils INFO Waiting for 2 seconds, retry every 1 seconds. (Function: kubernetes.dynamic.client.get Kwargs: {'field_selector': 'metadata.name==container-disk-vm-1693483207-3708382', 'namespace': 'virt-general-test-container-disk-vm'})
2023-08-31T15:03:03.483983 ocp_resources VirtualMachineInstance ERROR VirtualMachineInstance container-disk-vm-1693483207-3708382: status is None!/does not exist!
*** kubernetes.dynamic.exceptions.NotFoundError: 404
Reason: Not Found
HTTP response headers: HTTPHeaderDict({'Audit-Id': '31546098-5020-48dc-8a9b-ce54caa78728', 'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'Strict-Transport-Security': 'max-age=31536000; includeSubDomains; preload', 'X-Kubernetes-Pf-Flowschema-Uid': '7a14a742-a4ff-4502-8dc6-69b0943f1792', 'X-Kubernetes-Pf-Prioritylevel-Uid': '8cec7eb0-0f89-4eee-80af-f4d87bad54bd', 'Date': 'Thu, 31 Aug 2023 12:03:03 GMT', 'Content-Length': '312'})
HTTP response body: b'{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"virtualmachineinstances.kubevirt.io \\"container-disk-vm-1693483207-3708382\\" not found","reason":"NotFound","details":{"name":"container-disk-vm-1693483207-3708382","group":"kubevirt.io","kind":"virtualmachineinstances"},"code":404}\n'
Original traceback: 
...
```